### PR TITLE
Fixes #41 - Update contribution process and Remove internal tools.

### LINKED ISSUE #41: Proposal around PR / Merge process
So I was thinking about this a bit... What if when we merge PRs, instead of just pressing the button, we manually merge locally and as part of completing this merge, we increment the metadata in the fix version. ... But in that manner, we're "releasing" a new version with each PR merge, but we only upload to supermarket when we do a process release (and all the internal paperwork, to increment the enhancement version)? 

We could then make stable the main branch of the repository (since each checkout would be "releasable"... you might get newer features than what's in supermarket / signed off on, but since we'd have unique fix versions for each incremental change to stable, this isn't as much of a problem. Downloading master for the latest signed off version would still be an option, and when a new signed off version is released, the version number won't conflict. 

Thoughts? :+1: / :-1: ?

### PR DESCRIPTION
**Pull request by acharlieh**
Pull request for #41 where we remove the internal roll-out gem and its configurations, and update the contributing doc appropriately.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,25 +2,27 @@
 1. Cookbook related Issues should be logged on Github's issue tracker, and are considered to be the source of truth. 
     1. Issues will be triaged and marked with an appropriate label (e.g. bug, enhancement, question).
     2. A corresponding internal JIRA could be logged if there needs to be internal discussion or if an internal code review is desired.
-2.  The issue should be worked in a branch (Outside of the cerner/cerner_splunk repo)
+2. The issue should be worked in a branch (Outside of the cerner/cerner_splunk repo)
 3. The feature is pull requested to 'stable' 
-    1. The feature branch should use as few logical commits as possible with descriptive commit messages
-    2. Attach a pull request to the issue (using a tool like hub, github's ui doesn't let you do this but the API does) if it's a simple / obvious change, or create a new pull request referencing the issue if it's larger / complex. (judgement call).
+    1. The feature branch should use as few logical commits as possible with descriptive commit messages (These commits should not reference the issue, that will be done when it is merged)
```
Comment (garrypolley) [inline CONTRIBUTING.md]: I would bold or italicize the `not` portion. "These commits should _**not**_ ..."

**Comment by garrypolley** [comment]
:+1:

File: CONTRIBUTING.md (hunk 1 above)
Comment (acharlieh) [inline CONTRIBUTING.md]: Sounds good to me. Fixed.

**Comment by aaronblythe** [comment]
+1

**Comment by nimeshsubramanian** [comment]
+1

**Pull request merged**